packages/forms-web-app/src/dynamic-forms/validator/multifile-upload-validator.js

```diff
@@ -1,6 +1,8 @@
 const BaseValidator = require('./base-validator.js');
 
 const multifileUploadSchema = require('../../validators/common/schemas/multifile-upload-schema.js');
+// todo(journey-refactor): uses clamav, config etc, inject this?
```

Comment:
I think maybe we share some but not all of our validators? If it's a bring-your-own-questions type of affair then BO can plug in whatever they need at that point

Reply:
yes makes sense, although thinking we don't share the entire multifile upload component